src/pages/api/saves/index.ts

```diff
@@ -123,19 +129,22 @@ async function get(req: NextApiRequest, res: NextApiResponse) {
 }
 
 async function post(req: NextApiRequest, res: NextApiResponse) {
+  // console.log("Saving...");
+  // console.log(process.env.DATABASE_URL);
   const uid = await getUID(req, res);
   const players = JSON.parse(req.body) as Player[];
   for (const player of players) {
     try {
       const r = await conn.execute(
         `
-        REPLACE INTO Saves (_id, user_id, general, fishing, cooking, crafting, shipping, museum, social, monsters, walnuts, notes, scraps, perfection, powers)
-        VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
+        REPLACE INTO Saves (_id, user_id, general, bundles, fishing, cooking, crafting, shipping, museum, social, monsters, walnuts, notes, scraps, perfection, powers)
+        VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
       `,
         [
           player._id,
           uid,
           player.general ? JSON.stringify(player.general) : "{}",
+          player.bundles ? JSON.stringify(player.bundles) : "[]",
```

Comment:
was [] meant to be {}?

Reply:
iirc something with the schema required him to use array, no worries